packages/iTwinUI-react/stories/core/Table.stories.tsx

```diff
@@ -2349,6 +2349,73 @@ Virtualized.argTypes = {
   data: { control: { disable: true } },
 };
 
+export const ScrollToItem: Story<Partial<TableProps>> = (args) => {
```

Comment:
Would the name "Scroll To Row" make more sense here since it only applies to rows?